java/java-tests/testData/codeInsight/daemonCodeAnalyzer/quickFix/optionalIsPresent/afterAnnotation.java

```diff
@@ -1,4 +1,4 @@
-// "Replace Optional.isPresent() condition with functional style expression" "true-preview"
+// "Replace Optional presence condition with functional style expression ''Optional.ifPresent()''" "true"
```

Comment:
Double single quotes should not be used in messages.

Reply:
Fixed